manpages/lumina-fileinfo.8

```diff
@@ -0,0 +1,27 @@
+.Dd October 24, 2017
+.Dt LUMINA-FILEINFO 8
+.Os 1.3.3
+
+.Sh NAME
+.Nm lumina-fileinfo
+.Nd is used to view the properties of specific files. It also includes a simple XDG application registration creator.
+
+.Sh SYNOPSIS
+.Nm
+.Op file
+
+.Sh DESCRIPTION
+.Nm
+displays a graphical utility for viewing/changing the properties of a file. It yields
```

Comment:
Don't use symbols as shortcuts. They can be confusing to readers: "viewing/changing" -> ... displays a graphical utility to view and change file properties.

Reply:
"Yields" feels a little awkward -> It shows